lib/atmega328/rotary-encoder.cpp

```diff
@@ -59,14 +63,63 @@ ISR(INT0_vect)
     }
 }
 
+#define LONG_PRESS_DURATION 3000
+#define DOUBLE_PRESS_DURATION 500
```

Comment:
Prefer typed values before defines. That way you also get a reasonable compiler warning / error if you're doing something strange.

```suggestion
const uint16_t long_press_duration_ms = 3000;
const uint16_t double_press_duration_ms = 500;
```

Pre-processor statements can potentially give more headaches, and should not be resorted to unless necessary.

Reply:
I see. I will start using const then 😅. Change implemented.